### DISCUSSION TITLE
One page, multiple button？？？？？？.

### DISCUSSION THREAD

**Discussion by Chenjiujiu** [remix-run/remix#9622] [Q&A]
"In Remix, typically, one route is associated with one Action function. If my backend management system page contains over 20 operations buttons, each triggering different data requests or operations, how should I handle this?"

**Comment by gustavopch** [comment]
https://www.jacobparis.com/content/remix-multiple-actions
https://sergiodxa.com/articles/multiple-forms-per-route-in-remix#the-action-reducer-pattern
https://www.youtube.com/watch?v=w2i-9cYxSdc
https://github.com/remix-run/remix/discussions/3138
https://github.com/remix-run/remix/discussions/3898

**Comment by Erij-Maherzia-BEN-BRAHIM** [comment]
@Chenjiujiu you can use the named actions from `remix-utils`

[remix-utils/named-action](https://github.com/sergiodxa/remix-utils?tab=readme-ov-file#named-actions)
